packages/next/src/server/app-render/rsc/taint.ts

```diff
@@ -0,0 +1,12 @@
+/*
+
+Files in the rsc directory are meant to be packaged as part of the RSC graph using next-app-loader.
+
+*/
+
+import * as React from 'react'
+
+// @ts-ignore
+export const taintObjectReference = React.experimental_taintObjectReference
```

Comment:
@huozhi How can trick Webpack to let me import this only if it's available like when the taint flag is on?